src/index.js

```diff
@@ -118,14 +119,25 @@ class ReactTooltip extends Component {
     let targetArray = this.getTargetArray()
 
     let dataEvent
+    let dataEventOff
     for (let i = 0; i < targetArray.length; i++) {
       if (targetArray[i].getAttribute('currentItem') === null) {
         targetArray[i].setAttribute('currentItem', 'false')
       }
       dataEvent = this.state.event || targetArray[i].getAttribute('data-event')
       if (dataEvent) {
-        targetArray[i].removeEventListener(dataEvent, this.checkStatus)
-        targetArray[i].addEventListener(dataEvent, this.checkStatus, false)
+        dataEventOff = this.state.eventOff || targetArray[i].getAttribute('data-event-off');
+        // if off event is specified, we will show tip on data-event and hide it on data-event-off
+        if (dataEventOff) {
+          targetArray[i].removeEventListener(dataEvent, this.showTooltip);
```

Comment:
I haven't tested it thoroughly, but so far it works :-) To be honest, I don't fully understand the checkStatus method. Seems like it's there just to make sure that when the "event" happens, it closes the tooltip if it is opened, or opens it if it is closed, which you don't need when the opening/closing is bound to different events.